components/indexer.py

```diff
@@ -15,7 +15,7 @@ class Indexer:
     intake_left_motor: wpilib.interfaces.SpeedController  # Looking from behind the robot
     intake_right_motor: wpilib.interfaces.SpeedController  # Looking from behind the robot
 
-    indexer_speed = tunable(0.6)
+    indexer_speed = tunable(0.4)
```

Comment:
Was this intentional? What's the reason behind this change?

Reply:
I don't think this was intentional, will revert